.gitignore

```diff
@@ -11,3 +11,5 @@ coverage/
 
 .idea/
 
+.env
```

Comment:
that entry in the gitignore file could have slipped in because of me. I think we've been using an env file for a paypal API connection, but I would have to ask again

Reply:
This entry is intentional:
We have introduced commands in the bounded context that can work as standalone commands without needing to integrate with the FUN app (although part of https://phabricator.wikimedia.org/T340734 will be to integrate the commands into the FUN app as well). The commands need PayPal credentials and the most convenient and idiomatic way to provide those is through an `.env` file. Credentials in `.env` files must never be checked into the repo, so we added it to `.gitignore` as a small protection against that.

We'll explain the commands and the configuration in our further efforts in #131 (in the README).